bls/mcl.go

```diff
@@ -9,6 +9,9 @@ import "C"
 import "fmt"
 import "unsafe"
 
+// CurveSNARK1 --
```

Comment:
Probably worth adding here “curve for SNARKs; corresponds to the curve primitives on the Ethereum VM”.

Reply:
Or “original libsnark curve, ...”